packages/commons/src/AxeRunContext.cs

```diff
@@ -13,5 +14,60 @@ public class AxeRunContext
         public List<string[]> Include { get; set; }
         [JsonProperty("exclude")]
         public List<string[]> Exclude { get; set; }
+
+        /// <summary>
```

Comment:
I don't think we should include opaque axe-internal properties in the same user-facing type we encourage for use with `axe.run` parameters. Rather than adding all these new properties to the existing `AxeRunContext` type, I would lean towards just having `AxeFrameContext`'s `Context` property be an opaque `object` or `JObject` type and document on the property that it's intentionally opaque, only suitable to be passed to `axe.runPartial` as-is.